packages/devtools_app/lib/src/performance/performance_screen.dart

```diff
@@ -308,17 +309,32 @@ class TimelineConfigurationsDialog extends StatelessWidget {
     final theme = Theme.of(context);
 
     return DevToolsDialog(
-      title: dialogTitleText(theme, 'Recorded Streams'),
+      title: dialogTitleText(theme, 'Performance Settings'),
+      includeDivider: false,
       content: Container(
         width: dialogWidth,
         child: Column(
           mainAxisSize: MainAxisSize.min,
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
+            ...dialogSubHeader(theme, 'Default Recorded Timeline Streams'),
             ..._defaultRecordedStreams(theme),
             const SizedBox(height: denseSpacing),
-            ...dialogSubHeader(theme, 'Advanced'),
+            ...dialogSubHeader(theme, 'Advanced Recorded Timeline Streams'),
             ..._advancedStreams(theme),
+            const SizedBox(height: denseSpacing),
+            ...dialogSubHeader(theme, 'Additional Settings'),
+            Row(
+              children: [
+                NotifierCheckbox(notifier: controller.badgeTabForJankyFrames),
+                Text(
+                  'Badge Performance tab when Flutter UI jank is detected',
+                  // Specify font size to match style of other RichText widgets
```

Comment:
should we just use RichText instead of text to avoid this?
Hard coding font sizes will make it hard for us to later support custom font sizes for DevTools which I believe @helin24 was working on.

Reply:
Yes I'd like to add custom font sizes! (mainly for the intellij embedded browser) That change is stuck waiting for a refactor of some controllers though

Reply:
done